cmd/tailscale/cli/set.go

```diff
@@ -40,6 +40,35 @@ Only settings explicitly mentioned will be set. There are no default values.`,
 	UsageFunc: usageFuncNoDefaultValues,
 }
 
+// stringSliceMultiFlag is a flag.Value implementation that collects multiple
+// string values into a slice. It supports specifying the flag multiple times
+// as well as comma-separated values in each instance of the flag.
+//
+// For example, given a flag definition:
+//
+//	var inputs stringSliceMultiFlag
+//	flag.Var(&inputs, "input", "Comma-separated list of inputs; may be repeated")
+//
+// The following command-line arguments:
+//
+//	-input=foo -input=bar,baz
+//
+// Will result in:
+//
+//	inputs == []string{"foo", "bar", "baz"}
+type stringSliceMultiFlag []string
+
+func (s *stringSliceMultiFlag) String() string {
+	return strings.Join(*s, ",")
+}
+
+func (s *stringSliceMultiFlag) Set(v string) error {
+	for part := range strings.SplitSeq(v, ",") {
```

Comment:
If `v == ""`, this will append an empty string to the slice, we should probably special-case that.